src/list/views/CollectionView.ts

```diff
@@ -144,21 +149,21 @@ export default Marionette.PartialCollectionView.extend({
             return;
         }
         const firstChild = this.el.children[0];
-        if (!(firstChild && firstChild.classList.contains('row'))) {
+        if (!(firstChild && firstChild.tagName === 'TR')) { // TODO: remove?
             return;
         }
 
         let childHeight = firstChild.offsetHeight;
         if (!childHeight) {
-            const element = document.createElement('div');
-            element.innerHTML = firstChild.outerHTML;
+            const element = firstChild.cloneNode();
             document.body.appendChild(element);
             childHeight = element.offsetHeight;
             document.body.removeChild(element);
         }
         if (childHeight > 0) {
             this.childHeight = childHeight;
             this.__isChildHeightSpecified = true;
+            this.stopListening(this.collection.parentCollection, 'add remove reset ', this.__specifyChildHeight);
```

Comment:
this.stopListening(this.collection.parentCollection, this.__specifyChildHeight); ?